bson-scala/build.gradle

```diff
@@ -58,7 +58,10 @@ test {
     maxParallelForks = 1
 }
 
-jar.manifest.attributes['Import-Package'] =  [
-        '!scala.*',
-        '*'
-].join(',')
+afterEvaluate {
+    jar.manifest.attributes['Automatic-Module-Name'] = 'org.mongodb.scala.bson'
```

Comment:
This one seems inconsistent with the others, where `bson` is before the language. Should we change to org.mongodb.bson.scala?